### DISCUSSION TITLE
Column title with HTML

### DISCUSSION THREAD

**Discussion by dgillier** [rappasoft/laravel-livewire-tables#808] [Q&A]
Hello,

I'm new to this component and cannot find a way to have HTML in  a specific column (icon instead of text for example) ?

Any idea on how to do this ?
Thanks, Denis

**Comment by DaniWinter** [comment]
Would also like to know how to do this..

**Reply by lrljoe** [reply]
If it's a Column body that you want to do this in, then that's easily done using the format or label options.

For Column Titles, you can use setThAttributes in your configure method.  You can set custom attributes per-Column, to adjust this.

However, for v3, I will look at adding a method to the Column to make this more straight forward to achieve.

**Comment by lrljoe** [comment]
Closing as feature request exists for this. noting that you can presently do this with setThAttributes in configure()
